apps/web-giddh/src/app/reports/components/report-details-components/report.details.component.ts

```diff
@@ -293,4 +308,23 @@ export class ReportsDetailsComponent implements OnInit {
 
         return { firstDay, lastDay };
     }
+
+    /**
+     * Calculates the sales register total
+     *
+     * @private
+     * @param {*} transaction Sales transaction
+     * @memberof ReportsDetailsComponent
+     */
+    private setSalesRegisterTotal(transaction: any): void {
```

Comment:
if this code is used in multiple places then you can use this block of code in utility or service

Reply:
Actually the calculation varies based on the module therefore not included in utility service @rohit-walkover